src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -346,6 +347,10 @@ export class ExposureNotificationService {
     }
   }
 
+  async clearExposedStatus() {
+    return this.finalize({type: ExposureStatusType.Monitoring});
```

Comment:
By only calling `finalize`, is it possible that an artifact like `notificationSent` could remain in the `ExpsosureStatus` value? `notificationSent` would be set to a value when the status is `ExposureStatusType.Exposed`, but I can't find anything that removes that key from the status in `finalize()`.

Reply:
Good question 

Looking at where the code resets 

`updateExposure` 

After the `EXPOSURE_NOTIFICATION_CYCLE` the `notificationSent` etc.. doesn't get reset.

I'll dig on this a bit.